dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/features/ServerApi.java

```diff
@@ -61,7 +61,7 @@
 
 @RequestFilters({ BasicAuthentication.class, OrganisationIdFilter.class })
 @Consumes(MediaType.APPLICATION_JSON)
-@Path("/{jclouds.api-version}/server")
+@Path("/2.6/server")
```

Comment:
Change the default version in the API metadata instead of doing this. This way, if users want to use the old path they can just override the property when creating the context. All new fields in the Server object are optional, so that shouldn't be an issue.

Reply:
Do all APIs need to us ethe same version? If so, youd it make sense to store the version in a constant and reference it from all APIs?

Reply:
Well, we implement them in chunks. This PR covers only ServerApi feature which will be updated to new version. Everything else remain old 2.4 for now. We can introduce constant with newest API version supported, but outdated features will have to have 2.4 version hardcoded in them until we implement latest version of them. Would that be ok?

Reply:
Understood. Let's leave it as-is then for now.